massgov/mayflower
# DP-23086 A11y when org search is not shown, it should be fully hidden for all users

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description
Hides the search form so it's not focusable by the keyboard instead of using just opacity.

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-23086)

## Steps to Test


1. Visit any organization and place the focus on the organization navigation.
2. Press the "tab" key until focusing on the search, the next tab should go to the content and not dissapear.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
